src/hardware/imfc.cpp

```diff
@@ -13384,8 +13384,9 @@ static void imfc_init(Section* sec)
 	                                 ChannelFeature::ChorusSend,
 	                                 ChannelFeature::Synthesizer});
 
-	// Bring up the volume to get it on-par with line recordings
-	constexpr auto volume_scalar = 4.0f;
+	// Default volume scalar adjusted to match hardware line-out levels
+	// recorded by Pierre: Ref: https://www.youtube.com/watch?v=WHVWDi15AIw
+	constexpr auto volume_scalar = 2.1f;
```

Comment:
Can we get the emulation to generate a full-scale test signal? Like a 1kHz sine wave? It would be nice to also mention what the peak value of such a signal should be on the DBFS scale (e.g., -3.1dB).

https://en.m.wikipedia.org/wiki/DBFS

Reply:
That would be nice; although we'd probably want to run that on real hardware, too?  (I don't know how to do this though.. well, without having to learn the custom MIDI-like calls and going down a rabbit hole with it).

These games do generate tones across most of the frequency range (including percussion), and the 2.1 scalar was a very close approximation versus the hardware; and I checked quite a few. 

```
cap-1.90vol-441khz-nofilter-nomixer.wav
cap-2.00vol-48khz-nofilter-nomixer.wav
cap-2.10vol-44100-hz-filter-lpf-2-3500Hz-mixer-effects-off.wav
cap-2.13vol-48khz-nofilter-nomixer.wav
cap-2.15vol-441khz-nofilter-nomixer.wav
cap-2.17vol-48khz-nofilter-nomixer.wav
cap-2.20vol-48khz-filter-on-nomixer.wav
cap-2.25vol-48khz-filter-lpf-2-3500Hz-mixer-effects-off.wav
cap-2.27vol-441khz-nofilter-nomixer.wav
cap-2.30vol-48khz-nofilter-nomixer.wav
cap-2.42vol-48khz-filter-on-nomixer.wav
```

Reply:
> That would be nice; although we'd probably want to run that on real hardware, too? (I don't know how to do this though.. well, without having to learn the custom MIDI-like calls and going down a rabbit hole with it).

That's right, and even measuring the peak DBFS value of a test signal on the emulation side would require figuring out how to generate that signal in the first place. Maybe just leave it as is, then 😄